content/guides/references/trade-offs.md

```diff
@@ -37,7 +37,6 @@ Many of these issues are currently being worked on or are on our
 - [Workarounds for the lack of a `cy.hover()` command.](/api/commands/hover)
 - [`cy.tab()` command.](https://github.com/cypress-io/cypress/issues/299)
 - [There is not any native or mobile events support.](https://github.com/cypress-io/cypress/issues/311#issuecomment-339824191)
-- [Testing file uploads is application specific.](https://github.com/cypress-io/cypress/issues/170#issuecomment-340012621)
```

Comment:
This should have been removed back when I released `selectFile`, back late last year, but I didn't notice it at that point. 
We resolved the issue a year ago, just forgot to clean up a link in the docs. :woman_shrugging: 

It can be its own PR, but I hate the overhead of creating a branch, opening a PR with description and getting multiple reviewers for a one-line cleanup.